### PR TITLE
Fix returned data for already present floating IP

### PR DESCRIPTION
**Pull request by onurguzel**
##### SUMMARY
Floating IP module is idempotent when a Droplet ID is provided. However, the returned data structure is different between the run when the IP is created and the run it is already present. This pull-request fixes the returned data for the state that the floating IP is present. Please see the output below.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
`community.digitalocean.digital_ocean_floating_ip`

##### ADDITIONAL INFORMATION
The test:
```yaml
  - name: Create floating IP address
    community.digitalocean.digital_ocean_floating_ip:
      state: present
      droplet_id: "123456"
    register: floating_ip
  - debug:
      var: floating_ip
```

Output when floating IP is created:
```
TASK [Create floating IP address] **********************************************
changed: [localhost]

TASK [debug] *******************************************************************
ok: [localhost] => {
    "floating_ip": {
        "changed": true,
        "data": {
            "floating_ip": {
                "droplet": null,
                "ip": "12.34.56.78",
                "locked": true,
                "region": {
                    ... (omitted)
                }
            },
            "links": {}
        },
        "failed": false
    }
}

```

Output when floating IP is existing (before the PR):
```
TASK [Create floating IP address] **********************************************
ok: [localhost]

TASK [debug] *******************************************************************
ok: [localhost] => {
    "floating_ip": {
        "changed": false,
        "data": {
            "floating_ip": "12.34.56.78"
        },
        "failed": false
    }
}
```

Output when floating IP is existing (after the PR):
```
TASK [Create floating IP address] **********************************************
ok: [localhost]

TASK [debug] *******************************************************************
ok: [localhost] => {
    "floating_ip": {
        "changed": false,
        "data": {
            "floating_ip": {
                "droplet": {
                    ... (omitted)
                },
                "ip": "12.34.56.78",
                "locked": false,
                "region": {
                    ... (omitted)
                }
            }
        },
        "failed": false
    }
}
```

### CODE REVIEW

**Comment by mamercad** [comment]
Ah, thank you, please add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment).

**Review by mamercad — CHANGES_REQUESTED** [review]
Needs a changelog fragment.

**Comment by onurguzel** [comment]
Thank you for your quick response. I have added the fragment.

Would you be kind enough to include this fix in 1.10.0 version?

File: changelogs/fragments/162-floating-ip-data-idempotency.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - digital_ocean_floating_ip - make floating ip return data idempotent. (https://github.com/ansible-collections/community.digitalocean/pull/162)
```
Comment (mamercad) [inline changelogs/fragments/162-floating-ip-data-idempotency.yaml:2]: The period should be moved to after the reference URL, i.e.:

```yaml
bugfixes:
  - digital_ocean_floating_ip - make floating ip return data idempotent (https://github.com/ansible-collections/community.digitalocean/pull/162).
```

**Review by mamercad — CHANGES_REQUESTED** [review]

File: changelogs/fragments/162-floating-ip-data-idempotency.yaml (hunk 1 above)
Comment (onurguzel) [reply changelogs/fragments/162-floating-ip-data-idempotency.yaml]: Fixed it. Thanks for the guidance.

**Review by mamercad — APPROVED** [review]
LGTM

**Pull request merged**